docker-compose.yml

```diff
@@ -8,22 +8,22 @@ services:
       args:
         KOMODO_BRANCH: dev
     # image: dragonriders/kmdplatform_komodod_dev:latest
-    image: kmdplatform_komodod_dev
-    restart: on-failure
+    image: testkomodo
+    #restart: on-failure
     networks:
       - komodo_scale
     ports:
     #  - "32200:32200"
       - "32199:32199"
-    volumes:
```

Comment:
Please add all volumes back. What you need to do is to modify .env file and point to some directory on your disk.